docs/zh/latest/plugins/api-breaker.md

```diff
@@ -21,98 +21,69 @@ title: api-breaker
 #
 -->
 
-## 目录
+## 简介
 
-- [**定义**](#定义)
-- [**属性列表**](#属性列表)
-- [**启用方式**](#启用方式)
-- [**测试插件**](#测试插件)
-- [**禁用插件**](#禁用插件)
+启用该插件后，网关将根据配置判断上游是否异常，若异常，则直接返回预设的错误码，且在一定时间内不再访问上游。
 
-## 定义
+## 参数
 
-该插件实现 API 熔断功能，帮助我们保护上游业务服务。
+|        参数名称         |    类型    | 必选  | 默认值 | 使用范围  |                                        描述                                        |
+| :---------------------: | :--------: | :---: | :----: | :-------: | :--------------------------------------------------------------------------------: |
+|   break_response_code   |   整数型   |  是   |        | 200 ~ 599 |                           上游不健康时，将返回该状态码。                           |
```

Comment:
The indentation should be left alignment. The center alignment can't be maintained by humans. Please fix all the similar places.

Reply:
🤔 ok

Reply:
done